demo/index.html

```diff
@@ -151,6 +153,7 @@ <h1>Symphony Electron API Demo</h1>
                 activate: 'activate',
                 registerBoundsChange: 'register-bounds-change',
                 registerProtocolHandler: 'register-protocol-handler',
+                registerAnalyticHandler: 'register-analytic-handler',
```

Comment:
please refactor this to `registerAnalyticsHandler`: `register-analytics-handler`